components/datatypes/consentpreferences.schema.json

```diff
@@ -0,0 +1,337 @@
+            },
+            "xdm:eMail": {
+              "title": "Receive email",
+              "description": "User is willing to be contacted via email",
+              "$ref": "#/definitions/marketingOption"
+            },
+            "xdm:pushNotifications": {
+              "title": "Receive push notifications",
+              "description": "It is okay to send push notifications",
+              "$ref": "#/definitions/marketingOption"
+            },
+            "xdm:inAppMessages": {
+              "title": "Receive in-app messages",
+              "description": "User has allowed messages to be shown within the company's apps",
+              "$ref": "#/definitions/marketingOption"
+            },
+            "xdm:sms": {
+              "title": "Receive SMS",
+              "description": "It is okay to send text messages to the user",
+              "$ref": "#/definitions/marketingOption"
+            },
+            "xdm:phoneCalls": {
+              "title": "Receive phone calls",
+              "description": "The company may contact the user with phone solicitations",
+              "$ref": "#/definitions/marketingOption"
+            },
+            "xdm:snailMail": {
+              "title": "Receive snail-mail",
+              "description": "It is okay to send offers via traditional mail.",
+              "$ref": "#/definitions/marketingOption"
+            },
+            "xdm:inVehicleMessages": {
+              "title": "Receive in-vehicle messages",
+              "description": "It is okay to provide offers to the user via technology installed in their vehicle",
+              "$ref": "#/definitions/marketingOption"
+            },
+            "xdm:inHomeMessages": {
+              "title": "Receive in-home messages",
+              "description": "It is okay to provide offers to the user via technology installed in their home",
+              "$ref": "#/definitions/marketingOption"
+            },
+            "xdm:iot": {
+              "title": "Receive IoT messages",
+              "description": "It is okay to present offers via less traditional types of connected devices",
+              "$ref": "#/definitions/marketingOption"
+            },
+            "xdm:socialMedia": {
+              "title": "Receive Social Media Messages",
+              "description": "It is okay to direct messages to the user via their social media accounts",
+              "$ref": "#/definitions/marketingOption"
+            }
+          }
+        },
+        "xdm:timestamp": {
+          "title": "Consent Preferences timestamp",
+          "description": "Timestamp of the complete set of user consent preferences",
+          "type": "string",
+          "format": "date-time"
+        },
+        "xdm:version": {
+          "title": "Preferences Version",
+          "description": "Version of the Privacy Preferences Standard",
+          "type": "string"
+        },
+        "xdm:userLocale": {
```

Comment:
@RussStringham , Can you clarify what this field is for and how it relates or doesn't with the existing locale information? - https://github.com/adobe/xdm/blob/master/components/datatypes/placecontext.schema.json

Reply:
This is not related to language, but legal jurisdiction. If set to EU or a specific country in the UK, then GDPR would apply. If set to California (CA) then CCPA would apply. If set to US then CCPA might apply.

Reply:
Should the name of this field be changed from xdm:userLocale to xdm:jurisdiction?

Reply:
I like the idea of changing the name to reference a jurisdiction and being more concrete.  I would also change it to an enum with the jurisdictions we understand.